### PR TITLE
opensea.io-nftsalepage.com + more

### PR DESCRIPTION
**Pull request by 409H**
opensea.io-nftsalepage.com
Fake OpenSea site 
https://urlscan.io/result/b6b9c94a-3b4e-4129-b17a-2f4e13ca4df1/

extension-node.com
Fake WalletConnect site
https://urlscan.io/result/54d617b0-2fd2-4b47-a227-cab8a6eb293a/

metamaskwebx.com
Fake MetaMask site phishing for secrets with POST //nuisance-play.herokuapp.com/api/v1/phrases
https://urlscan.io/result/5132273b-4c89-4410-9e03-598776584ab2/
https://urlscan.io/result/1a45ce31-ce27-4ca9-8cf7-97ebe28dd422/

polygon.tecnologiy.org
Fake MetaMask site phishing for secrets with POST /metamask.php
https://urlscan.io/result/5f180adb-5e0a-4461-988a-f2738ee5afd9/

eversgrowcoin.com
Fake MetaMask site phishing for secrets with POST /login-error.php
https://urlscan.io/result/90813331-520e-4f90-8771-0ce784d088fd/
https://urlscan.io/result/5c2b11e8-3263-400a-b5e1-b5cebde42f70/

metamask-extension.com.hsurge.com
Fake MetaMask site phishing for secrets with POST /token.php
https://urlscan.io/result/b4b20e8e-681b-49c4-9258-43ecfb399fb4/

meta-mask-wallet-security.web.app
Fake MetaMask site phishing for secrets with POST //app-4b11c7cb-d09e-4ad5-bc3e-b34e7deb1623.cleverapps.io/metamask_logs
https://urlscan.io/result/955dd541-5f98-4bb3-9193-4b821520f7cc/

boasecure-verify.duckdns.org
Fake MetaMask site phishing for secrets with POST /getData.php
https://urlscan.io/result/9e553c22-aba0-45cf-bc4d-9e3beaadd2c5/

othersidemeta-nft.app
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/18280dba-e657-4661-8f2d-9321921518b2/
address: 0x2567921B42DFA7788cf5Df251c4c4278C281D33A (eth)

othiersidemeta-refund.xyz
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/1a3309ca-da65-4bc1-a20f-8aeeae9a5f5b/
address: 0xBE82Eb372612379572a4772C6A1a27a402ce3C1a (eth)

othersiderefund.xyz
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/869cc7ce-0266-4a40-9817-2fca720e8044/

refund-othersidemeta.xyz
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/5509da24-a140-4c7b-82a2-686e440a46a2/

othersidemeta-claim.xyz
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/e9b607a6-b932-4422-97f0-a39e88e351de/
address: 0x6F7A49Da0F184509814039956137dadb9ccda4f8 (eth)

othersidemeta-refund.xyz
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/ac249a1b-d6a2-426d-b802-7f30ab2b91fa/
address: 0x6F7A49Da0F184509814039956137dadb9ccda4f8 (eth)

gasrefund-otherside.com
Fake BoredApeYachtClub NFT site phishing for funds
https://urlscan.io/result/ea9e385e-155d-4e8e-a70f-edf7ab924983/
address: 0x6F7A49Da0F184509814039956137dadb9ccda4f8 (eth)

metamasks.ca
Fake MetaMask site phishing for secrets with POST /wp-admin/admin-ajax.php
https://urlscan.io/result/2a3c1d98-8284-4c3d-a820-d3e9ea4f5f2a/
https://urlscan.io/result/4acb8ea0-e469-4715-9c5d-7563a426e8b2/

walletconnectlive.crystalmining.live
Fake WalletConnect site phishing for secrets with POST /Otyzt.php
https://urlscan.io/result/6e5a5730-aca5-4320-bc46-dc69ddb87fb8/
https://urlscan.io/result/e9fbecb7-5195-42e0-af74-62ebc0255bdd/

magic-walletconnect.com
Fake MagicEden site phishing for secrets with POST /node/form.php
https://urlscan.io/result/aeaf82ef-e56e-41ef-a58d-cf198e93ba81/

uniswap-vvip.com
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/657a2a88-89dc-412f-8e73-451b7a5810c5/
address: 0xDEf9591B2895c3B1Cb8904752FB1B4802875Ba77 (eth)

bancor-vip.com
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/66e1582e-661f-4e0e-a16c-5aecb12c2921/
address: 0x818af6a967FB329C04abC9D9E2301483cc268636 (eth)

eth-waet.com
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/142e7482-7e3c-4d48-9ce0-e497844cb0d9/
address: 0xf86395bd1de4164cb30c90171000596fad4c4ff9 (eth)

dpethmin.me
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/dddcf099-5417-410e-ab09-88079bd5892b/
address: 0xb45263de3990d74661b13e050fd06fb36958b566 (eth)

dpethmin.com
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/05c1cee1-13ed-4196-b7ca-f56929c8ef19/
address: 0xb45263de3990d74661b13e050fd06fb36958b566 (eth)

metamask2022.io
Trust trading scam site
https://urlscan.io/result/6971b849-3da5-412a-a657-fe2cc8b57c9e/
address: 0x45D1cDa4f0BFb62e425edb3047532fCa5417da14 (eth)

get-ethereumrewards.rf.gd
Fake MetaMask site phishing for secrets with POST /login.php
https://urlscan.io/result/5a7b1dd7-11c6-44bc-8975-7ddc7477039c/
https://urlscan.io/result/d51ec538-f659-460b-af90-2f7d88bcd296/
https://urlscan.io/result/5ff767a9-5447-49bd-88f3-26fb5913d0d5/
https://urlscan.io/result/823c2d50-9e67-4b28-891a-088a57513901/
https://urlscan.io/result/ce2f0f97-f693-4c01-8303-19e4915a1e98/

etherbunny.mintbuy.me
Fake EtherBunny NFT site phishing for funds
https://urlscan.io/result/7198c2dd-ce77-42bf-b5f6-9074d8c3b0d8/
address: 0xa2BD0E425AD20bb0A024ee3Ea1E4A4947652dACA (eth)

boredape-yacht.com
Trust trading scam site
https://urlscan.io/result/ed82a2aa-d21a-48f5-be42-1ceca8c7e905/
address: 0xeCB41C3ffC8382478d8A18f438F16E517A4363bb (eth)

metamask-io.quickdiate.com
Fake MetaMask site 
https://urlscan.io/result/20e64a14-5e59-4b58-9b6b-07f22041ff90/

mining.coinbaseusdt.com.cn
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/4a5e8c6d-35f5-4eca-979c-bd95f588a8f1/
address: 0xa3163d5155db415b0ae08b0870d8f2d6413092dd (eth)

nodeminingpool.com
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/be9cd6ac-77f3-4006-b91c-d74736c30502/
address: 0x0052A8400f51a26A53DFc589745bc0DD18be4993 (eth)

mining-dapps.biz
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/ad209fec-683a-4246-8f7f-d089d17cd2c7/
address: 0x1d952b5fce42d50799be240cdd791978dc236c21 (eth)

ethswap.site
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/2c8c6c74-f443-468c-af3c-558c4ab1b15d/

coinbasepool.net
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/b2312450-1db7-4acb-8744-993fa48e44ab/
address: 0xe5b59bd05b61e280b49c13811abe7a423a807cc8 (eth)

cryptonvest.com
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/16a2de6d-b7c6-45a0-8739-70ea77a16c6d/
address: 0xb5590ce74a6efaef0a9080dd2c5e66ce3e143103 (eth)

restoremetamask.io
Fake MetaMask site phishing for secrets with POST /restore/handler.php
https://urlscan.io/result/7bf87bba-8aec-44b4-8a14-ec2aa5930b89/
https://urlscan.io/result/37238c64-3714-4084-ad34-bfe5f9503872/

pooltogether.top
Fake PoolTogether site phishing for secrets with POST /send.php
https://urlscan.io/result/a82742b0-29ad-4526-a17a-e2b2642c6e5b/
https://urlscan.io/result/56e7d42a-02b8-4b4b-b70f-b90d667f16cb/

restoremetamask.help
Fake MetaMask site phishing for secrets with POST /restore/handler.php
https://urlscan.io/result/261e1e37-518d-411b-bcc8-c9e14f4ce454/
https://urlscan.io/result/8570c7db-1447-4bd9-87bb-eea957f7f340/

verify-recoveryphrase.com
Fake MetaMask site phishing for secrets with POST /process.php
https://urlscan.io/result/50126db5-5acc-45eb-a179-1884c37ff68b/
https://urlscan.io/result/5b1a50a0-ffc6-4c78-9063-09a189884643/

metamask-wallet-security.com
Fake MetaMask site phishing for secrets with POST /
https://urlscan.io/result/682c465c-83a6-4ed5-93a3-6a2914b55942/

coinbase-usd.org
Fake investment platform phishing for funds - usdt approve abuse scam
https://urlscan.io/result/161b9440-ccff-48fe-9674-813785d676ab/
address: 0x769219aFB84F7940446A26143E434d04dC435167 (eth)

opensearewards-app.online
Fake OpenSea site phishing for secrets with POST /metamask/login.php
https://urlscan.io/result/45e3b736-446d-495b-8cc9-61e7fb6578e0/
https://urlscan.io/result/3f2c1959-a5e2-475f-8b9c-acdeafbc3052/
https://urlscan.io/result/320d27df-57d3-4f2f-9b92-d85dc8cebe9a/

bc1qh2p9ex5k3.com
Fake WalletConnect site phishing for secrets with POST /start/config/mega/import1.php
https://urlscan.io/result/f468aa15-d29e-4d47-abcf-c1f79c306361/
https://urlscan.io/result/f84acb43-b63c-4811-8f4d-46ec98e34ad4/

metamask.secure-confirm.com
Fake MetaMask site phishing for secrets with POST /
https://urlscan.io/result/1eafd08d-84aa-4c7a-8091-447f65bc9f42/

metamask.ddns.net
Fake MetaMask site phishing for secrets with POST /system/send_Phrase.php
https://urlscan.io/result/bed99809-9f76-45c2-90dc-5ccdd6df44f1/
https://urlscan.io/result/51269dd5-2a17-4ed9-9069-7c95ead67e72/

metamask-verifyprocess.net
Fake MetaMask site phishing for secrets with POST /
https://urlscan.io/result/0872a892-9bc5-4c0e-b0d6-e877d7dfd56a/

metamask.io.educapp.pe
Fake MetaMask site phishing for secrets with POST /
https://urlscan.io/result/c437aab8-8b07-4824-a142-9c27b9450b88/
https://urlscan.io/result/b35959c2-f059-4d4c-92ff-7104c9b99252/

### CODE REVIEW

**Review by AlexHerman1 — APPROVED** [review]
LGTM.
